### PR TITLE
Docs schedules

### PR DESCRIPTION
**Pull request by tpdorsey**
## Summary
Adds RRuleClock and example to Schedules documentation.

## Changes
On https://docs.prefect.io/core/concepts/schedules.html
- Add section on RRuleClock including example
- Add links for clocks to API docs
- Add note about use of pendulum so imports in examples are not overlooked, with link to pendulum site for further information

On https://docs.prefect.io/api/latest/schedules/clocks.html#rruleclock
- Fix docstring formatting and add context to links for iCal RFC and dateutil rrule module

This PR also adds a "Verified" logo image asset that in future will be used for task listings in environments that need an image, such as README files. Making the image assets publicly available now enables easier testing for future README updates regardless of github branch or repo a contributor may be rendering on.

## Importance
RRuleClock was not documented outside of API docs.

## Checklist

This PR:

~~- [ ] adds new tests (if appropriate)~~
~~- [ ] adds a change file in the `changes/` directory (if appropriate)~~
~~- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)~~

### CODE REVIEW

**Review by cicdw — DISMISSED** [review]
Awesome, thanks @tpdorsey this LGTM -- will leave open in case others have any comments

File: src/prefect/schedules/clocks.py [hunk 1]
```diff
@@ -390,8 +390,10 @@ class RRuleClock(Clock):
     Clock that supports iCal style RRule (recurrence rules).
 
     See below links for helpful info:
-        - https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html
-        - https://dateutil.readthedocs.io/en/stable/rrule.html
+    
+    IETF iCalendar specification RFC5545: https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html
+    
+    dateutil rrule module, an implementation of the iCal specification recurrence rules: https://dateutil.readthedocs.io/en/stable/rrule.html
```
Comment (cicdw) [inline src/prefect/schedules/clocks.py:396]: actually it looks like this broke `black` somehow - you'll need to run `black` to resolve

Comment (tpdorsey) [reply src/prefect/schedules/clocks.py:396]: Fixed.

**Review by anna-geller — APPROVED** [review]
Nice, thanks for adding the example!

**Pull request merged**